src/index.html

```diff
@@ -217,7 +217,7 @@ <h1>COVID-19 Spread Simulator 🦠</h1>
   </main>
   
   <footer>
-    Based on <a target="_blank" rel="noopener nofollow" href='https://www.washingtonpost.com/graphics/2020/world/corona-simulator/'>Washington Post article</a> | <a href='https://github.com/midudev/covid-19-spread-simulator' target="_blank" rel="noopener nofollow">Source Code</a> | By <a href='https://midu.dev'>@midudev</a> | Extended by <a href='https://www.linkedin.com/in/marco-pierobon-a82ab646/'>Marco Pierobon</a>
+    Based on <a target="_blank" rel="noopener nofollow" href='https://www.washingtonpost.com/graphics/2020/world/corona-simulator/'>Washington Post article</a> | <a href='https://github.com/noiapp/covid-19-spread-simulator' target="_blank" rel="noopener nofollow">Source Code</a> | By <a href='https://midu.dev'>@midudev</a>	    Based on <a target="_blank" rel="noopener nofollow" href='https://www.washingtonpost.com/graphics/2020/world/corona-simulator/'>Washington Post article</a> | <a href='https://github.com/midudev/covid-19-spread-simulator' target="_blank" rel="noopener nofollow">Source Code</a> | By <a href='https://midu.dev'>@midudev</a> 
```

Comment:
Duplicated "Based on ... | By ..." on this line.

Reply:
Yes, I had seen this and fixed it locally

Reply:
Change moved forward to next PR.